documentation/nbvalNotebook.py

```diff
@@ -46,6 +45,8 @@ def runAll():
             return None
 
 
+# pylint is not realizing the finally clause is important.
+# pylint: disable=try-except-raise
```

Comment:
I think pylint is just saying you can use try/finally instead of try/except/finally.

Reply:
Ah!  I did not realize you could use finally without except!